stl/src/msvcp_atomic_wait.src

```diff
@@ -35,6 +35,7 @@ EXPORTS
     __std_tzdb_delete_time_zones
     __std_tzdb_get_current_zone
     __std_tzdb_get_leap_seconds
+    __std_tzdb_get_sys_info_v2

```

Comment:
This is an ABI break - we can't change the export surface of DLLs, including adding any exports (for tedious reasons). That's why we had to add the (now-misnamed) atomic_wait satellite DLL in the first place, because we couldn't add exports directly to msvcp140.dll.

Yet another satellite DLL independently loading ICU would be an extremely logistically complicated change for a performance improvement. (Our policy is essentially to avoid satellite DLLs unless they are absolutely required to implement features; a perf improvement doesn't meet that bar.) Having the import lib independently load ICU would be less logistically complicated, but might not be worth it.

Can you think of any other way to achieve this without breaking ABI? If not, we'll have to close this PR and record the possible improvement as a vNext issue. Crying cat emoji.

Reply:
It seems possible to encode the type in the `_Tz` string, so this optimization (hopefully) doesn't need to wait for vNext.

Reply:
Thanks, after a brief glance this appears to be ABI-compatible (I'll need to think about it more, of course).